### PR TITLE
Docs/local time

### LINKED ISSUE #89: Document time zone handling in code and documentation
See issue #9 -- at this point : document that times in the netcdf files are local times for all grid cells and make sure that that is clearly stated in the metadata in the NetCDF files.

### PR DESCRIPTION
**Pull request by arbennett**
- [x] closes #89 
 - [x] tests added / passed
 - [x] passes ``git diff upstream/master | flake8 --diff``

Adds some documentation about the way that time works in MetSim's output.  See #89 for more discussion.

### CODE REVIEW

File: docs/data.rst [hunk 1]
```diff
@@ -54,6 +54,10 @@ grid cell. It should be named ``swe`` in the file.
 
 Output Specifications
 =====================
+.. ATTENTION::
+    The ``time`` coordinate in MetSim's output is local to the location of each 
+    cell!
+
```
Comment (bartnijssen) [inline docs/data.rst]: Perhaps expand to:

+.. ATTENTION::
 +    The ``time`` coordinate in MetSim's output is local to the location of each 
 +    cell. This means that for a single time slice in the NetCDF file all locations 
 +    along a parallel (same latitude) will have the same solar geometry at that time.

File: metsim/metsim.py [hunk 2]
```diff
@@ -86,11 +86,14 @@
                        'standard_name': 'relative_humidity'},
          'spec_humid': {'units': '', 'long_name': 'specific humidity',
                         'standard_name': 'specific_humidity'},
+         'time': {'long_name': 'local time at grid location',
```
Comment (bartnijssen) [inline metsim/metsim.py]: Does `time` have a units attribute or is that added already by `xarray`

Comment (arbennett) [inline metsim/metsim.py]: Yes, the the units come from pandas internally.

File: docs/data.rst [hunk 3]
```diff
@@ -66,9 +71,9 @@ step:
 * ``t_min`` : Minimum temperature (also a required input value) (C)
 * ``t_max`` : Maximum temperature (also a required input value) (C)
 * ``prec`` : Precipitation (also a required input value) (mm/day)
-* ``swe`` : Snow water equivalent (mm)
+* ``shortwave`` : Snow water equivalent (mm)
```
Comment (bartnijssen) [inline docs/data.rst]: Whoops - typo!

**Pull request merged**